gluoncv/auto/estimators/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Estimator implementations"""
-from .yolo import YoloEstimator
-from .center_net import CenterNetEstimator
+# from .yolo import YoloEstimator
```

Comment:
why disable all imports for estimators?

Reply:
import estimators here will report an error, but directly import from the inner files when using these estimators (e.g. in test_auto_detection.py) can avoid this error, this bug has not been fixed so I just comment them here.